custom/ewsghana/tasks.py

```diff
@@ -189,6 +190,13 @@ def migrate_email_settings(domain):
     endpoint = GhanaEndpoint.from_config(config)
     migrate_email = EmailSettingsSync(domain)
 
+    for user in iter_docs(WebUser.get_db(), WebUser.ids_by_domain(domain)):
+        for notification in ReportNotification.by_domain_and_owner(domain, user['_id'], stale=False):
+            notification.delete()
```

Comment:
is that the only issue? I thought another issue was that some reports are using archived locations in the filter. is that true, and are there any other issues besides those two?